lib/ui/painting/image_encoding.cc

```diff
@@ -132,19 +143,20 @@ void ConvertImageToRaster(sk_sp<SkImage> image,
   // to prevent concurrent usage of the image on both the IO and raster threads.
   raster_task_runner->PostTask([image, encode_task = std::move(encode_task),
                                 resource_context, snapshot_delegate,
-                                io_task_runner]() {
+                                io_task_runner, &gpu_disable_sync_switch]() {
```

Comment:
This should be capture a copy of the shared_ptr, not a reference to the shared_ptr.

Reply:
Done